npm/css-src/sass/_drag-and-drop.scss

```diff
@@ -2,31 +2,53 @@
   margin-bottom: 40px;
 }
 
+.drag-and-drop__button {
+  margin-bottom: 32px;
+  margin-top: 65px;
+  padding-left: 15px;
+  position: static;
```

Comment:
This might be my CSS ignorance, but why is this line needed? I thought `static` was the default?

Reply:
Yes, `static` is the default but for GOV.UK buttons, it is set to `relative` and that (for some reason) was causing the folder to open in a new tab when you dropped one on it; `static` was one of the only ones that didn't cause this behaviour.